src/debug.ts

```diff
@@ -0,0 +1,51 @@
+/*
+  TODO:
+  import DFP from './networks/DFP';
+*/
+
+import Ad from './Ad';
+import Page from './Page';
+import AutoRefresh from './plugins/AutoRefresh';
+import AutoRender from './plugins/AutoRender';
+import DeveloperTools from './plugins/DeveloperTools';
+import Responsive from './plugins/Responsive';
+import Sticky from './plugins/Sticky';
+import { IPlugin } from './types';
+
+const plugins: { [key: string]: IPlugin } = {
+  AutoRefresh,
+  AutoRender,
+  DeveloperTools,
+  Responsive,
+  Sticky,
+};
+
+const pluginKeys = Object.keys(plugins);
+
+pluginKeys.forEach((plugin) => {
+  if (!(window as any)[`_ADJS`].Plugins[plugin]) {
+
+    // If user doesn't include plugin in configuration, we add it here.
+    if (plugin === 'DeveloperTools') {
+      (window as any)[`_ADJS`].Plugins.DeveloperTools = DeveloperTools;
+    }
+
+    return;
+  }
+
+  Object.getOwnPropertyNames(plugins[plugin].prototype).forEach((method) => {
```

Comment:
This should be checking the plugins on the window not all plugins. I.e. for each on  _AdJS.Plugins to ensure you’re only loading plugins they’ve already loaded. Otherwise this will error